config/install/field.field.media.gallery.field_slide.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_slide
+    - media_entity.bundle.gallery
+    - media_entity.bundle.image
+    - media_entity.bundle.video
+id: media.gallery.field_slide
+field_name: field_slide
+entity_type: media
+bundle: gallery
+label: Slide
+description: ''
```

Comment:
Is "Slide" really correct label? There are many slides in this field. We should also add a description.